packages/editor/src/components/blocks/textarea/Textarea.tsx

```diff
@@ -0,0 +1,65 @@
+import type { Textarea, Prettify } from '@axonivy/form-editor-protocol';
+import type { ComponentConfig, UiComponentProps } from '../../../types/config';
+import './Textarea.css';
+import { baseComponentFields, defaultBaseComponent } from '../base';
+import IconSvg from './Textarea.svg?react';
+import { useEffect, useRef, useState } from 'react';
+
+type TextareaProps = Prettify<Textarea>;
+
+export const defaultInputProps: Textarea = {
+  label: 'Label',
+  value: '',
+  rows: '5',
+  ...defaultBaseComponent
+} as const;
+
+export const TextareaComponent: ComponentConfig<TextareaProps> = {
+  name: 'Textarea',
+  category: 'Elements',
+  subcategory: 'Input',
+  icon: <IconSvg />,
+  description: 'A simple and customizable Textarea component',
+  defaultProps: defaultInputProps,
+  render: props => <UiBlock {...props} />,
+  create: ({ label, value, ...defaultProps }) => ({ ...defaultInputProps, label, value, ...defaultProps }),
+  outlineInfo: component => component.label,
+  fields: {
+    label: { subsection: 'General', label: 'Label', type: 'text' },
+    value: { subsection: 'General', label: 'Value', type: 'textBrowser', options: { onlyTypesOf: 'Date' } },
+    rows: { subsection: 'General', label: 'Visible Rows', type: 'number' },
+    ...baseComponentFields
+  }
+};
+
+const UiBlock = ({ label, value, rows }: UiComponentProps<TextareaProps>) => {
+  const [height, setHeight] = useState('auto');
+  const contentRef = useRef<HTMLSpanElement>(null);
+
+  useEffect(() => {
```

Comment:
I made the placeholder textarea autoResizable, because the PrimeFace-Component is also autoResizable.

Reply:
Ok makes sense to show this somehow here, but not like this...
I even think, that the row count makes no sense to increase the height of the UI block...

This in an input block, so most of the time this will only have a data binding on it (#{data.text}). So this logic will never be triggered at all and if someone wants to have 10 rows then this block is huge, but the content is only this binding? I think this is a really bad UX...
Better would be something like a indicator, maybe on the right side how many rows and if auto resize is on (by the way, autoresize is also a property so I think this should be a property here too)

Reply:
Thank you, yes you're right. That really doesn't make much sense. So how about I leave the height always the same, display the row count at the top right and use a resize icon in the input field to indicate whether autoResize is active or not (if active, the icon is not displayed).
![grafik](https://github.com/user-attachments/assets/84dbe77b-c39c-4ddf-8094-c1b2b5ec3f80)

Reply:
yes better I think 👍 
maybe only rows? or better `5 rows`?